.github/workflows/rust.yml

```diff
@@ -0,0 +1,24 @@
+name: Rust PR CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --verbose --release
+    - name: Run tests
+      run: cargo test --verbose
```

Comment:
Do we need to run these in each directory still?
https://github.com/aws-cloudformation/cloudformation-guard/pull/64/files

Reply:
i set this up as a rust workspace so it builds everything in the directory (including sub directories). check the logs for one of the operating systems below to see that the project is building all the packages